Aptoide/aptoide-client-v8
# [ASV-1084] task: remove test fairy flavour.

**What does this PR do?**

   To remove everything related to test fairy.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] build.gradle.java (app)
- [ ] EntryPointChooser.java

**How should this be manually tested?**

  I would suggest running app in different flavours to see if it doesn't break in runtime during launch.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1084](https://aptoide.atlassian.net/browse/ASV-1084)

**Code Review Checklist**
